modules/mongodb/templates/mongodb-restore-s3.erb

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -e
+
+AWS_ACCESS_KEY_ID=<%= @aws_access_key_id %>
+AWS_SECRET_ACCESS_KEY=<%= @aws_secret_access_key %>
+S3_BUCKET=<%= @s3_bucket %>
+BACKUP_DIR=<%= @backup_dir %>
+
+cd $BACKUP_DIR
+
+
+TIME="$(date +%s)"
+
+s3cmd get --force `s3cmd ls s3://${S3_BUCKET}/mongodump* | tail -1 | awk '{print $4}'`| \
```

Comment:
Listing the contents of the bucket each time might become a slow operation when the bucket gets larger. An alternative method would be to know the name of the file you're GETing before you've even gone into the bucket i.e. "mongodump-latest.tar.gz"